prusti-tests/tests/verify/pass/arrays/selection_sort.rs

```diff
@@ -0,0 +1,56 @@
+use prusti_contracts::*;
+
+fn main() {}
+
+
+// TODO: replace 10 with a.len()
```

Comment:
Is this something that will be part of this PR?

Reply:
that's a relict from me writing the test before #538 was merged, and now that i checked, it'll also need more slice snapshot support to use in the invariants (unsize cast, `.len()` call, ..). For the non-pure part (i.e. the loop conditions) we can use `.len()` already, so i'll change that.